test/serializers/attributes_test.rb

```diff
@@ -23,12 +23,6 @@ def test_attributes_with_fields_option
                      @profile_serializer.attributes(fields: [:name]))
       end
 
-      def test_required_fields
-        assert_equal({name: 'Name 1', description: 'Description 1'},
-                     @profile_serializer.attributes(fields: [:name, :description], required_fields: [:name]))
-
-      end
-
```

Comment:
Despite of felling okay removing this, it just remembers me how badly we need the filter method hahah because handle with different set `fields` using the same serializer is a need.
Btw as I commented [here](https://github.com/rails-api/active_model_serializers/pull/1103/files#r38414212), it seems we are adding some new behavior, would nice to have it tested :smile: